kernel/classes/datatypes/ezimage/ezimagetype.php

```diff
@@ -254,6 +258,27 @@ function validateObjectAttributeHTTPInput( $http, $base, $contentObjectAttribute
                 'The size of the uploaded file exceeds the limit set for this site: %1 bytes.' ), $maxSize );
             return eZInputValidator::STATE_INVALID;
         }
+
+        // Check for a valid alternative text if there is an image and
+        // if the alt text is required
+        if(    $contentObjectAttribute->attribute( 'has_content' ) && self::isAltTextRequired( $contentObjectAttribute ) )
```

Comment:
I fixed my issues here.  There are a few instances left that were not part of my changes.  Should I change those as well?